zerver/views/users.py

```diff
@@ -115,8 +115,9 @@ def deactivate_user_backend(
     user_profile: UserProfile,
     *,
     user_id: PathOnly[int],
-    deactivation_notification_comment: Annotated[str, StringConstraints(max_length=2000)]
-    | None = None,
+    deactivation_notification_comment: (
+        Annotated[str, StringConstraints(max_length=2000)] | None
+    ) = None,
```

Comment:
Hmm why is this line changed? 
Have you fetch upstream and rebase upstream/main ?